repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/json/JsonSerializer.java

```diff
@@ -310,6 +332,10 @@ public void serializeData(ProjectInspector inspector) {
             case PATCHED:
                 outputHasBeenPatchedInspector(inspector);
                 break;
+                
+            case FIXERBUILD:
+            	outputFixerBuildInspector(inspector);
```

Comment:
Your choice: you can also create another serializer class inheriting from AbstractDataSerializer.